x-pack/test/functional/page_objects/space_selector_page.ts

```diff
@@ -212,7 +212,21 @@ export class SpaceSelectorPageObject extends FtrService {
     await this.testSubjects.setValue('descriptionSpaceText', descriptionSpace);
   }
 
+  async clickSwitchSpaceButton(spaceName: string) {
+    const collapsedButtonSelector = '[data-test-subj=euiCollapsedItemActionsButton]';
+    // open context menu
+    await this.find.clickByCssSelector(`#${spaceName}-actions ${collapsedButtonSelector}`);
+    // click context menu item
+    await this.find.clickByCssSelector(
+      `.euiContextMenuItem[data-test-subj="${spaceName}-switchSpace"]` // can not use testSubj: multiple elements exist with the same data-test-subj
```

Comment:
I couldn't see a way to click the link in the context menu simply using the data-test-subj, since multiple elements existed with the same attribute (the link text and the button). So I needed to use a more specific selector that involves the EUI class name.